src/uu/cp/src/cp.rs

```diff
@@ -290,179 +290,187 @@ static DEFAULT_ATTRIBUTES: &[Attribute] = &[
         .arg(Arg::with_name(OPT_ARCHIVE)
-             .short("a")
-             .long(OPT_ARCHIVE)
-             .conflicts_with_all(&[OPT_PRESERVE_DEFAULT_ATTRIBUTES, OPT_PRESERVE, OPT_NO_PRESERVE])
-             .help("Same as -dR --preserve=all"))
+            .short("a")
+            .long(OPT_ARCHIVE)
+            .conflicts_with_all(&[OPT_PRESERVE_DEFAULT_ATTRIBUTES, OPT_PRESERVE, OPT_NO_PRESERVE])
+            .help("Same as -dR --preserve=all"))
         .arg(Arg::with_name(OPT_NO_DEREFERENCE_PRESERVE_LINKS)
-             .short("d")
-             .help("same as --no-dereference --preserve=links"))
+            .short("d")
+            .help("same as --no-dereference --preserve=links"))
         .arg(Arg::with_name(OPT_ONE_FILE_SYSTEM)
-             .short("x")
-             .long(OPT_ONE_FILE_SYSTEM)
-             .help("stay on this file system"))
+            .short("x")
+            .long(OPT_ONE_FILE_SYSTEM)
+            .help("stay on this file system"))
 
         // TODO: implement the following args
         .arg(Arg::with_name(OPT_COPY_CONTENTS)
-             .long(OPT_COPY_CONTENTS)
-             .conflicts_with(OPT_ATTRIBUTES_ONLY)
-             .help("NotImplemented: copy contents of special files when recursive"))
+            .long(OPT_COPY_CONTENTS)
+            .conflicts_with(OPT_ATTRIBUTES_ONLY)
+            .help("NotImplemented: copy contents of special files when recursive"))
         .arg(Arg::with_name(OPT_SPARSE)
-             .long(OPT_SPARSE)
-             .takes_value(true)
-             .value_name("WHEN")
-             .help("NotImplemented: control creation of sparse files. See below"))
+            .long(OPT_SPARSE)
+            .takes_value(true)
+            .value_name("WHEN")
+            .help("NotImplemented: control creation of sparse files. See below"))
         .arg(Arg::with_name(OPT_CONTEXT)
-             .long(OPT_CONTEXT)
-             .takes_value(true)
-             .value_name("CTX")
-             .help("NotImplemented: set SELinux security context of destination file to default type"))
+            .long(OPT_CONTEXT)
+            .takes_value(true)
+            .value_name("CTX")
+            .help("NotImplemented: set SELinux security context of destination file to default type"))
         .arg(Arg::with_name(OPT_CLI_SYMBOLIC_LINKS)
-             .short("H")
-             .help("NotImplemented: follow command-line symbolic links in SOURCE"))
+            .short("H")
+            .help("NotImplemented: follow command-line symbolic links in SOURCE"))
         // END TODO
 
         .arg(Arg::with_name(OPT_PATHS)
-             .multiple(true))
+            .multiple(true))
+}
+
+pub fn uumain(args: impl uucore::Args) -> i32 {
+    let usage = get_usage();
+    let matches = uu_app()
+        .after_help(&*format!(
+            "{}\n{}",
+            LONG_HELP,
+            backup_control::BACKUP_CONTROL_LONG_HELP
+        ))
+        .usage(&usage[..])
```

Comment:
Lifetimes :( we create the usage String, but clap only takes a &str, so it would not live long enough if we included it. I'd love to use something like https://crates.io/crates/const_format in the future to make the usage string a constant, then it could work.

Reply:
I should've known, it's always lifelimes :) Thanks!